src/data-workspace/data-details-sidebar/comment.js

```diff
@@ -153,6 +195,15 @@ export default function Comment({ item }) {
         deId: item.dataElement,
         cocId: item.categoryOptionCombo,
     })
+    const contextSelectionId = useContextSelectionId()
+    const commentId = getCellId({ contextSelectionId, item })
+    const unsavedComment = useUnsavedDataStore((state) =>
```

Comment:
`CommentEditForm` is already importing the store (for the setter), so I would argue this can be moved down to that component. 
Or move the setter to this component.